src/QirRuntime/lib/QIR/arrays.cpp

```diff
@@ -261,10 +290,18 @@ extern "C"
         return array->dimensionSizes[dim];
     }
 
-    QirArray* quantum__rt__array_copy(QirArray* array)
+    QirArray* quantum__rt__array_copy(QirArray* array, bool force)

```

Comment:
In order to make the meaning of `force` more explicit, would it make sense to rename it to `forceNewInstance` or something similar?

Reply:
Done!